anesthetic/samples.py

```diff
@@ -80,16 +80,8 @@ def __init__(self, *args, **kwargs):
             self.limits = kwargs.pop('limits', {})
             self.label = kwargs.pop('label', None)
             self.root = None
-            self.u = None
-
             super(MCMCSamples, self).__init__(*args, **kwargs)
 
-            self.u = numpy.random.rand(len(self))
-
-            if self.w is not None:
-                self['weight'] = self.w
-                self.tex['weight'] = r'MCMC weight'
```

Comment:
I think it's worth keeping this, it's nice to have this in the table when printing `samples`...